lib/async_hooks.js

```diff
@@ -60,19 +60,11 @@ async_wrap.setupHooks({ init,
                         destroy: emitDestroyN });
 
 // Used to fatally abort the process if a callback throws.
-function fatalError(e) {
```

Comment:
Calling `FatalException()` bypasses any chance for Promise or try/catch to intercept the error and allows the application to exit either normally or abort if `--abort-on-uncaught-exception` was passed.